lib/sparse/src/index/search_context.rs

```diff
@@ -130,24 +139,31 @@ impl<'a> SearchContext<'a> {
     /// b,  21, 34, 60, 200
     /// b,  30, 34, 60, 230
     fn advance(&mut self) -> Option<ScoredPointOffset> {
-        let min_record_id = Self::next_min_id(&self.postings_iterators)?;
```

Comment:
Because there is only one usage of `next_min_id` left, WDYT about `next_min_id` signature refactoring?

Reply:
What do you in mind exactly?
FYI the next PR is removing this feature completely :broom:

Reply:
Great! 🧹🧹🧹